tests/integration/fns.sh

```
@@ -21,10 +21,10 @@ CIPHERNODE_SECRET="We are the music makers and we are the dreamers of the dreams
 
 # These contracts are based on the deterministic order of hardhat deploy
 # We _may_ wish to get these off the hardhat environment somehow?
-ENCLAVE_CONTRACT="0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512"
-REGISTRY_CONTRACT="0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9"
-REGISTRY_FILTER_CONTRACT="0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9"
-INPUT_VALIDATOR_CONTRACT="0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6"
+ENCLAVE_CONTRACT="0x8A791620dd6260079BF849Dc5567aDC3F2FdC318"
```

Shellcheck (0.11.0)

[warning] ENCLAVE_CONTRACT appears unused. Verify use (or export if used externally).

(SC2034)